src/pkg/reg/adapter/helmhub/consts.go

```diff
@@ -1,7 +1,6 @@
 package helmhub
 
 import "fmt"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380